spec/i18n_spec.rb

```diff
@@ -3,10 +3,45 @@
 require 'i18n/tasks'
 require 'yaml_normalizer'
 
+ALLOWED_UNTRANSLATED_KEYS = [
+  'account.navigation.menu', # "Menu" is "Menu" in French
+  'doc_auth.headings.photo', # "Photo" is "Photo" in French
```

Comment:
should we find a way to add locale-specific exceptions? Ex we wouldn't want this to accidentally allow copy-pasting English to Spanish, just because French and English match?

```
{ key: 'account.navigation.menu', locales: %i[fr] },
{ key: 'doc_auth.headings.photo', locales: %i[fr] },
```

Reply:
Yeah, that does seem like it could be a very easy thing to overlook. I can see what it might take to incorporate this locale-specific checking.

Reply:
Updated in 6639291.